### ISSUE TITLE
Documentation error in infinite-scroll usage for Ionic 4 beta documentation

### ISSUE THREAD

**Issue by codyburleson** [ionic-team/ionic#15510]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)          : 4.1.2 (/usr/local/lib/node_modules/ionic)
   Ionic Framework            : @ionic/angular 4.0.0-beta.7
   @angular-devkit/core       : 0.7.5
   @angular-devkit/schematics : 0.7.5
   @angular/cli               : 6.1.5
   @ionic/ng-toolkit          : 1.0.8
   @ionic/schematics-angular  : 1.0.6

System:

   NodeJS : v8.11.3 (/usr/local/bin/node)
   npm    : 6.4.1
   OS     : macOS High Sierra
```

**Describe the Bug**

There is a documentation error on the following page in the usage section for infinite-scroll component:

https://beta.ionicframework.com/docs/api/infinite-scroll#usage

The TypeScript example currently shows the following:

```
loadData(event) {
    setTimeout(() => {
      console.log('Done');
      event.target.complete();

      // App logic to determine if all data is loaded
      // and disable the infinite scroll
      if (data.length == 1000) {
        event.target.disabled = true;
      }
    }, 500);
  }
```

With the example given, `event.target.complete()` is not a function. Since we have the infinite scroll reference because of `@ViewChild(InfiniteScroll) infiniteScroll: InfiniteScroll;`, the line should be:

`this.infiniteScroll.complete();`

So, in other words, change this:

`event.target.complete();`

to this...

`this.infiniteScroll.complete();`

I assume that should also apply for the line `event.target.disabled = true;`. That should be `this.infiniteScroll.disabled = true;` 

**Steps to Reproduce**

Recreate the example shown in a new Ionic template project.

**Additional Context**

Of course, when you have it correct in the working code behind the interactive example in the documentation. It's just not correct in the TypeScript example of the Usage section.
